source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -241,8 +241,8 @@ unittest
     network.generateBlocks(Height(GenesisValidatorCycle - 3));
 
     // Make 2 nodes sleep for longer than test can possibly run
-    node_4.ctrl.sleep(180.seconds, true);
-    node_5.ctrl.sleep(180.seconds, true);
+    node_4.ctrl.sleep(1.hours, true);
```

Comment:
Is there any reason for `1.hours` instead of a dozen of minutes?

Reply:
Not really, but it will be cancelled when `sleep(0)` is called later so it could be `1.year` even.